packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -340,6 +344,96 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
     }
   }, [isVirtualized, scrollRef, manager.focusedKey]);
 
+  let prevScroll = useRef({scrollTop: scrollRef?.current?.scrollTop || 0, scrollLeft: scrollRef?.current?.scrollLeft || 0});
+  // Bring focused key into view if focus enters the collection from outside. Only for non virtualized collections,
+  // virtualized collections should handle this in their virtualizer or get this for free via our Virtualizer component
+  useEffect(() => {
+    let onFocus = (e) => {
```

Comment:
Problem with this approach: if the user focuses a cell -> scrolls so that it is out of view -> moves focus out of the table -> clicks on the table column header -> the table's scrollable region will scroll to bring the previous cell into view. This happens because this capturing listener triggers before useGridCell/useSelectableItem's onFocus can make the column header the new focusedKey, thus this block thinks it needs to scroll the old cell into view.
 
Not sure what to do here